app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,34 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = current_user.recipes.where(user_id: params[:food_id])
+  end
+
+  def new
+    @recipe = current_user.recipes.new(user_id: params[:food_id])
+  end
```

Comment:
- [ ] This method is used to create an instance of a new Recipe model.

Kindly use the snippet below:

def new
    @recipe = Recipe.new
  end